integration-tests/utils/integration-utils.js

```diff
@@ -29,37 +29,39 @@ const findInGrid = async (page, text) => {
   return await page.waitForXPath(`//*[@role="grid"][contains(normalize-space(.),"${text}")]`)
 }
 
-const exactlyFindClickable = (page, text) => {
-  return page.waitForXPath(`(//a | //*[@role="button"] | //button)[normalize-space(.)="${text}"]`)
+const clickable = ({ text, textContains }) => {
+  const base = '(//a | //*[@role="button"] | //button)'
+  if (text) {
```

Comment:
Did you intentionally choose to favor exact text over partial match in the case that both are specified? I think I agree with that choice.

TOL: An alternative would be something like `{ text, partialMatch = false}` to make self-contradictory parameters impossible.

Reply:
They're intended to be mutually exclusive, so perhaps we should assert that. Also, there may very well be other non-exclusive options here, e.g. `disabled: false`